install.md

```diff
@@ -121,7 +121,8 @@ Then to install Buildah on Ubuntu follow the steps in this example:
   export GOPATH=`pwd`
   git clone https://github.com/projectatomic/buildah ./src/github.com/projectatomic/buildah
   cd ./src/github.com/projectatomic/buildah
-  PATH=/usr/lib/go-1.8/bin:$PATH make runc all TAGS="apparmor seccomp"
+  PATH=/usr/lib/go-1.8/bin:$PATH
```

Comment:
This does not look correct.  The first version is correct since it will set the environment for PATH when running the make command.